mimic/model/identity_objects.py

```diff
@@ -62,17 +80,164 @@ def not_found(message, request):
     return _identity_error_message("itemNotFound", message, NOT_FOUND, request)
 
 
-def forbidden(message, request):
+def unauthorized(message, request):
     """
-    Return a 403 error body associated with a Identity forbidden error.
+    Return a 401 error body associated with a Identity unauthoriazed error.
     Also sets the response code on the request.
 
     :param str message: The message to include in the bad request body.
     :param request: The request on which to set the response code.
 
     :return: dictionary representing the error body.
     """
-    return _identity_error_message("forbidden", message, FORBIDDEN, request)
+    return _identity_error_message("unauthorized", message, UNAUTHORIZED, request)
+
+
+@implementer(IEndpointTemplate, IPlugin)
+class EndpointTemplateStore(object):
+    """
+    A :obj"`EndpointTemplateStore` is an object whiich provides the
```

Comment:
"provides the functionality to integrate" is pretty maddeningly vague.  I realize that we're a couple of meta-levels out here from the actual functionality but can you say what it does more directly?